site/en/Test.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+lang-ref: first test
+lang: en
+---
+
+# Results and findings of the first OCR-D test
+## Background
+At the turn of the year 2019/2020, the OCR-D software was tested for the first time in nine pilot libraries. This was to ensure the practical acceptance of the software by future, potential users. Therefore the focus was on its functionality and usability in practice. In addition to the partners of the coordination project, two libraries involved in the module projects as well as four other libraries took part in the testing. The findings of this first test run will be incorporated into the further development of the OCR-D prototype.
+
+All pilot libraries have some know-how and experience with OCR, as they have already produced full texts at least at project level or via service providers. The extent to which OCR, which is regarded as important, should be carried out independently in future and firmly anchored in the digitisation workflow, is currently discussed in the libraries. Concerning the target groups of the full texts there are different opinions in the testing institutions. While one third generally name humanities scholars, another third would like to serve a very broad target group (humanities, digital humanities, computer linguistics and economics). The remaining libraries only see relatively few, specialised users (Digital Humanities or Computational Linguistics) as the target group for OCR texts.
+
+The pilot libraries demand the following features of an OCR software:
+* High recognition rate of layout and text
+* Cost-effective use
+* Quick adaptability/troubleshooting
+* Modularity
+* Output in standard formats
+* Connection to existing workflows
+* Well documented interfaces
+* Word coordinates
+* Trainability
+* Extensive GT corpora
+
+High quality of text recognition is most important, whereas the other characteristics are only named each by some of the pilot libraries and should be considered as desired but subordinate optional features.
+
+
+## Evaluation of the software tests
+In order to ensure the comparability of the individual tests in the pilot libraries, a questionnaire distributed to the pilot libraries at the beginning of the test. In this questionnaire, the general conditions of the test run, e.g. the technical equipment used and the tested OCR-D processors, as well as the documentation of the software, interfaces, functionality and usability of the software, its possibilities for being integrated into existing workflows and the required output formats are recorded. Furthermore, recognition quality, functionality and usability, open requirements and positive features of the OCR-D software as well as the results of the test were to be described.
```

Comment:
of the test -> of the test phase

tested OCR-D processors -> OCR-D processors tested

are recorded -> In this questionnaire we asked about 

 were to be described ->  Furthermore, participants were asked to describe ...